applications_tests/dem_3d/periodic_boundary_gmsh.output

```diff
@@ -0,0 +1,24 @@
+Running on 1 rank(s)
+*****************************************************************
+
+DEM time-step is 7.71088% of Rayleigh time step
+Reading triangulation
+
+Finished reading triangulation
+
+Warning: There are diamond-shaped cells in the input triangulation. It is strongly recommended to use a different triangulation without such cells. It should be mentioned that these cells are not detected if you have grid motion
+Warning: expansion of particle-wall contact list is disabled.
```

Comment:
expansion of particle should be on since your geometry gmsh is a pipe

Reply:
This is the feature that is currently not working with periodic or outlet boundaries with DEM unfortunately. Will work on that in next PR.